dll/win32/setupapi/cfgmgr.c

```diff
@@ -1078,6 +1078,9 @@ CM_Add_Range(
 
     }
 
+    /* If we're done then free allocated resource in the memory */
+    HeapFree(GetProcessHeap(), 0, pRange);
```

Comment:
Following @ThFabba's comment, this HeapFree should be in the "else" block of above; also, please add an `UNIMPLEMENTED;` call too.